hugo.toml

```
@@ -138,11 +138,11 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.27"
+latest = "v1.28"
 
 version = "v1.27"
-githubbranch = "main"
-docsbranch = "main"
+githubbranch = "v1.27.4"
+docsbranch = "release-1.27"
 deprecated = false
```

Shouldn't deprecated=true here per [the handbook's example](https://github.com/kubernetes/website/pull/27455/files)?

---

@divya-mohan0209 is correct 
```suggestion
deprecated = true
```

---

I apologize for missing this earlier 😓